packages/edit-site/src/components/global-styles/font-library-modal/style.scss

```diff
@@ -154,7 +145,3 @@ button.font-library-modal__upload-area {
 		max-width: 350px;
 	}
 }
-
-.font-library-modal__font-collection__notice {
-	margin: 0;
-}
```

Comment:
The Notice component's margins have now been removed, so we no longer need to reset them.